src/header/js/header-menus.js

```diff
@@ -265,19 +307,31 @@ export default ({variant = 'desktop'}) => {
 						))
 				);
 			}
-			$items.push(
-				$('li', {class: `${prefix}-subitem`},
-					$('a', {href: entry.href, class: `${prefix}-sublink`},
-						$('p', {class: `${prefix}-sublink--title`},
-							entry.label
-						),
-						$('p', {class: `${prefix}-sublink--description`},
-							entry.description
+			
+			if (entry.href && entry.label && entry.description) {
+				$items.push(
+					$items.push(
```

Comment:
Why these two `items.push statements`? Do you want to get an array composed of `[item, array.length, ...]`? $items.push returns the length of the array.